Pipfile

```diff
@@ -23,4 +23,4 @@ tests = "bash ./run-tests.sh"
 
 [pipenv]
 
-allow_prereleases = true
\ No newline at end of file
+allow_prereleases = true
```

Comment:
Is this still needed after upgrading to sphinx-click 1.1.0?

Reply:
@uranusjr not related to this PR but is black still in prerelease? If so then this line is necessary if you want to install white & have black work properly